### PR TITLE
Fix/ui adapter metrics

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Discover metrics for adapters per run 
- Poll for metrics only after they are emitted.
- Minor styling fixes for metrics table.

### CODE REVIEW

File: cdap-ui/app/features/adapters/templates/tabs/runs/tabs/status.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <my-plumb data-is-disabled="true" ng-if="nodes.length > 0" data-reloaddag="true" data-config="nodes"></my-plumb>
 
-  <table class="table table-responsive adapter-metrics-table" ng-if="isMetrics">
+  <table class="table table-responsive table-bordered adapter-metrics-table" ng-if="isMetrics">
```
Comment (elfenheart) [inline cdap-ui/app/features/adapters/templates/tabs/runs/tabs/status.html]: table-responsive is supposed to be applied as a div wrapping the table

**Comment by ajainarayanan** [comment]
@elfenheart Updated.

File: cdap-ui/app/features/adapters/controllers/tabs/runs/tabs/status-ctrl.js [hunk 2]
```diff
@@ -77,26 +81,22 @@ angular.module(PKG.name + '.feature.adapters')
...
+      function onMetricsDiscoverySuccess(res) {
+        widget.metric.names = res;
+        if (res.length > 0) {
+          FetchMetricsData(widget);
+        } else {
+          $scope.formattedData = [];
+        }
+      }
+    );
+    function FetchMetricsData(widget) {
```
Comment (elfenheart) [inline cdap-ui/app/features/adapters/controllers/tabs/runs/tabs/status-ctrl.js]: Any reason why this function starts with capital letter? We usually reserve capital letter for Objects/Factories right?

**Comment by elfenheart** [comment]
minor comments, otherwise :+1: LGTM

**Comment by ajainarayanan** [comment]
Fixed the function rename, hence merging.

**Pull request merged**